File: app/src/main/java/com/SmartWatchVoice/bestapp/utils/Logger.java
package com.SmartWatchVoice.bestapp.utils;

import android.util.Log;

public class Logger {
    private static final String Tag = "TouchAlexa";
    public static void d(String msg) {
        Log.d(Tag, msg);
    }

    public static void i(String msg) {
        Log.i(Tag, msg);
    }

    public static void w(String msg) {
        Log.w(Tag, msg);
    }

    public static void e(String msg) {
        Log.e(Tag, msg);
    }

    public static void v(String msg) {
        Log.v(Tag, "[=======]" + msg);
    }
}

File: SmartWatchSDK/src/main/java/jie/android/alexahelper/smartwatchsdk/utils/Logger.kt
package jie.android.alexahelper.smartwatchsdk.utils

import android.util.Log

object Logger {
    private const val TAG: String = "AlexaHelper"
    public fun d(msg: String): Unit {
        Log.d(TAG, msg)
    }
    public fun i(msg: String): Unit {
        Log.i(TAG, msg)
    }
    public fun w(msg: String): Unit {
        Log.w(TAG, msg)
    }
    public fun e(msg: String): Unit {
        Log.e(TAG, msg)
    }
    public fun v(msg: String): Unit {
        Log.v(TAG, "[======]$msg")
    }
}
